tests/integration_tests/saas/test_mixpanel_task.py

```diff
@@ -0,0 +1,154 @@
+import random
+import time
+
+import pytest
+from fidesops.core.config import config
+from fidesops.graph.graph import DatasetGraph
+from fidesops.models.privacy_request import PrivacyRequest
+from fidesops.schemas.redis_cache import PrivacyRequestIdentity
+from fidesops.task import graph_task
+from fidesops.task.filter_results import filter_data_categories
+from fidesops.task.graph_task import get_cached_data_for_erasures
+from tests.graph.graph_test_util import assert_rows_match
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_mixpanel
+def test_mixpanel_saas_access_request_task(
+    db,
+    policy,
+    mixpanel_connection_config,
+    mixpanel_dataset_config,
+    mixpanel_create_test_data,
+) -> None:
+    """Full access request based on the Mixpanel SaaS config"""
+
+    email = mixpanel_create_test_data
+
+    privacy_request = PrivacyRequest(
+        id=f"test_saas_access_request_task_{random.randint(0, 1000)}"
+    )
+    identity = PrivacyRequestIdentity(**{"email": email})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = mixpanel_connection_config.get_saas_config().fides_key
+    merged_graph = mixpanel_dataset_config.get_graph()
+    graph = DatasetGraph(merged_graph)
+
+    time.sleep(15)
```

Comment:
Take a look at this file to see how we handle waiting for data to be available

https://github.com/ethyca/fidesops/pull/473/files#diff-3cda79699fd4f98cf63ed2fde3dfc22fce51b924b91a8167b9d4346f916bc3b9R192

The fixed sleeps tend to not be very reliable.